docs/cunumeric/source/index.rst

```diff
@@ -1,19 +1,39 @@
+:html_theme.sidebar_secondary.remove:
+
 Welcome to cuNumeric's documentation!
-=========================================
+=====================================
+
+cuNumeric is a `Legate`_ library that aims to provide a distributed and
+accelerated drop-in replacement for the `NumPy API`_ on top of the `Legion`_
+runtime.
+
+Using cuNumeric you do things like run the final example of the
+`Python CFD course`_ completely unmodified on 2048 A100 GPUs in a
+`DGX SuperPOD`_ and achieve good weak scaling.
 
 .. toctree::
   :maxdepth: 2
+  :caption: Contents:
 
-  Overview <README.md>
-  Build instructions <BUILD.md>
-  API Reference <api/index.rst>
-  API coverage <comparison/comparison.rst>
-  Contributing <CONTRIBUTING.md>
-  Versions <versions.rst>
+  user/index
+  comparison/index
+  api/index
+  developer/index
+
+.. toctree::
+  :maxdepth: 1
+
+  versions
 
 
 Indices and tables
-==================
+------------------
 
 * :ref:`genindex`
 * :ref:`search`
+
+.. _DGX SuperPOD: https://github.com/barbagroup/CFDPython/blob/master/lessons/15_Step_12.ipynb
```

Comment:
This link sounds wrong

Reply:
In what way? I took it from the readme and it seems to go to the right notebook

Reply:
I agree but that is what it is in the README where I copied from w/o paying close attention apparently. What is the best product link to use?

Reply:
https://www.nvidia.com/en-us/data-center/dgx-superpod/ ?

Reply:
https://github.com/nv-legate/cunumeric/pull/805/commits/b393a112496883d47dba96cf786fd10c8ace63b1

Reply:
LOL. Yes, that's a good link. Do you mind updating the README as well? 🤦

Reply:
https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0 (both)

Reply:
well I guess that's not the right commit link,  but both are updated.